### DISCUSSION TITLE
Please change duplicate to auto selected new duplicated drawing

### DISCUSSION THREAD

**Discussion by nitzanwilnai** [excalidraw/excalidraw#8920] [General]
Drives me crazy when I duplicate a drawing, that it doesn't automatically select the duplication. 

Let's say I want to take an existing drawing an change it:
1. I select the drawing, lets say "drawing 1".
2. I choose duplicate and name the new drawing "drawing 2".
3. I start modifying the drawing, thinking I am on "drawing 2" but I am actually still on "drawing 1"!

Please change it so after duplication it selects the new drawing.

**Comment by Mrazator** [comment]
Can you elaborate a bit? Notice the following example, when the duplicated element is auto-selected.

https://github.com/user-attachments/assets/9226e8b6-af07-4237-8363-be7fc20432b2

**Comment by nitzanwilnai** [comment]
Not the elements, the scenes. It should autoselect the scene Drawing 2.


https://github.com/user-attachments/assets/a85e410d-9750-483b-b1a6-05a67c97a963

**Reply by Mrazator** [reply]
I see, makes sense, wasn't sure what "drawing" meant.

Pinging E+ team, cc @maielo

**Reply by maielo** [reply]
Thanks, @Mrazator & @nitzanwilnai makes sense! Adding to our queue!
